data/docs/instrumentation/manual-instrumentation/javascript/nodejs.mdx

```diff
@@ -0,0 +1,157 @@
+- **Resource Utilization**: To monitor and manage resource usage, such as memory and CPU consumption.
+- **Business KPIs**: For capturing key performance indicators relevant to your business logic.
+
+### Set Up Custom Spans
+
+#### Install Dependencies
+
+If you haven't already, install the necessary OpenTelemetry libraries:
+
+```bash
+npm install @opentelemetry/api @opentelemetry/sdk-node @opentelemetry/resources @opentelemetry/semantic-conventions
+```
+
+#### Acquiring a Tracer
+
+Acquire a tracer to create spans:
+
+```jsx
+import { trace } from '@opentelemetry/api'
+const tracer = trace.getTracer('your-service-name')
+```
+
+#### Adding Custom Spans
+
+Example of adding a custom span in a function:
+
+```jsx
+async function yourFunction() {
+  return tracer.startActiveSpan('your-span-name', async (span) => {
+    try {
+      // Start a new active span named 'your-span-name'
+      span.addEvent('Event description') // Add an event with a description to the span
+      span.setAttribute('attributeKey', 'attributeValue') // Set an attribute on the span
+      span.setStatus({ code: SpanStatusCode.OK }) // Set the status of the span to OK
+    } catch (error) {
+      span.setStatus({ code: SpanStatusCode.ERROR, message: error.message }) // Set the status to ERROR if an exception occurs
+      span.recordException(error) // Record the exception in the span
+      throw error
+    } finally {
+      span.end() // End the span
+    }
+  })
+}
+```
+
+#### Running the Application
+
+Run your application normally, ensuring the tracing configuration is loaded first.
+
+#### Verifying the Manual Spans
+
+Check SigNoz for the manually created spans and verify the attributes and events are correctly logged.
+
+For more detailed instructions with a [sample app](https://github.com/SigNoz/opentelemetry-nodejs-example), refer to the complete article in the series: [Manual Instrumentation for Traces - OpenTelemetry NodeJS](https://signoz.io/opentelemetry/add-manual-span-to-traces-nodejs/)
+
+### Set Up Custom Metrics
+
+#### Setup Metric Collection
+
+Create a `telemetry.js` file and configure the `MeterProvider` with an `OTLPMetricExporter`:
+
+```jsx
+import { metrics } from '@opentelemetry/api'
+import {
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  ConsoleMetricExporter,
+} from '@opentelemetry/sdk-metrics'
+import { Resource } from '@opentelemetry/resources'
+import {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+} from '@opentelemetry/semantic-conventions'
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http'
+
+// Define resource with service name
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'your-service-name',
+  })
+)
+
+// Configure OTLP metrics exporter
+const otlpMetricsExporter = new OTLPMetricExporter({
+  url: 'http://otel-collector:4318/v1/metrics', // ensure the port is correct
```

Comment:
Can you please explain after the code snippet what `otel-collector` is in the URL endpoint